src/modules/lua.c

```diff
@@ -58,6 +58,79 @@ static pthread_mutex_t mtev_lua_states_lock = PTHREAD_MUTEX_INITIALIZER;
 static mtev_hash_table mtev_coros;
 static pthread_mutex_t coro_lock = PTHREAD_MUTEX_INITIALIZER;
 
+struct lua_module_gc_params {
+  int iters_since_full;
+  int full_every;
+  int steps;
+  int steps_multiplier, set_steps_multiplier;
+  int pause_size, set_pause_size;
```

Comment:
Your commit comment (below) is helpful to interpret the members of this struct. It would be helpful if it the comment would be somewhere in the source. Maybe here is a good place?
 
```
gc_full: n = 0 no full, n > 0 full every n iterations
gc_step: arg to lua_gc LUA_GCSTEP
gc_stepmul: arg to lua_gc LUA_GCSETSTEPMUL
gc_pause: arg to lua_gc LUA_GCSETPAUSE (% growth)
```

Reply:
I strongly dislike docs in two places.  I've added the parameter documentation for the modules which gets pulled into the main published docs.